### ISSUE TITLE
Mobile- Agreement camp name missing and then the design is looking awkward when subscribed 

### ISSUE THREAD

**Issue by saniatg** [the-canonizer/Canonizer-Beta--Issue-Tracking#1267]
![ImportedPhoto 747552544 316991](https://github.com/user-attachments/assets/1033514a-f1d9-45ce-ad24-d96cf090ada0)

**Comment by ks-rahul** [comment]
@saniatg, a New design has been implemented for breadcrumbs. Please check this once and let me know if you still need any changes.

**Comment by sajid315** [comment]
@saniatg seems like the reported issue is on older design of breadcrumb. 
Recently we updated the breadcrumb design and now that issue is not coming any-more as you can see in below screenshot.

![image](https://github.com/user-attachments/assets/2fa15e3a-6373-46a1-89f6-13f43a66d201)

@ks-rahul I just noticed one layout thing that we must fix, little unnecessary margin is coming on top of breadcrumb in mobile that is due to header layout. Can you please look into this only ? After that we can move this ticket ahead.

**Note:** This top margin is not coming on eventline page, we need to check it.

**Comment by ks-rahul** [comment]
@sajid315 is it right?

![Image](https://github.com/user-attachments/assets/f06ad301-d4f1-48ac-bba7-4415c634ef2f)

**Comment by sajid315** [comment]
> @sajid315 is it right?
> 
> ![Image](https://private-user-images.githubusercontent.com/95217406/391850731-f06ad301-d4f1-48ac-bba7-4415c634ef2f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMyMTQ2NTQsIm5iZiI6MTczMzIxNDM1NCwicGF0aCI6Ii85NTIxNzQwNi8zOTE4NTA3MzEtZjA2YWQzMDEtZDRmMS00OGFjLWJiYTctNDQxNWM2MzRlZjJmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjAzVDA4MjU1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI4YzYxODA2NWNmMTZkNTA3ZDE2ZjRlOWYzNjYzNzIxMWNiNjc5YjMxZjE2YTQ1NmVmNjk3NTQ2MmZkMWJkZTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.VZa1909hSGVFy9Mwn-pXgPJ6msZnrautHUvHUeM2QwU)

Yes @ks-rahul, it's OK for me. Further more QA team can confirm for all pages. Thanks

**Comment by saniatg** [comment]
Working fine on dev

**Comment by hassaanCD** [comment]
Verified on live
